clang/include/clang/Basic/arm_sve.td

```diff
@@ -1341,6 +1341,26 @@ def SVWHILEHS_U32 : SInst<"svwhilege_{d}[_{1}]", "Pmm", "PUcPUsPUiPUl", MergeNon
 def SVWHILEHS_U64 : SInst<"svwhilege_{d}[_{1}]", "Pnn", "PUcPUsPUiPUl", MergeNone, "aarch64_sve_whilehs", [IsOverloadWhile]>;
 }
 
+let TargetGuard = "sve2p1|sme2"  in {
```

Comment:
I would suggest to remove sme2 or add the flag here and in the tests.

Reply:
I've added a test run with "sme2".

Reply:
So that is the problem. It should not work  with sme2 when we do not have the attribute __arm_streaming_compatible . If we check the ACLE these new functions needs it:
  svboolx2_t svwhilege_b8[_s64]_x2(int64_t rn, int64_t rm)
    __arm_streaming_compatible;
I believe this is under work atm